tox.ini

```diff
@@ -0,0 +1,10 @@
+[tox]
+envlist = py3
+
+[testenv]
+deps =
+  coverage==4.5.4
+commands =
+  sh scripts/update_assets.sh
+  sh scripts/update_assets.sh
```

Comment:
A comment is needed here that second update_assets call is not a typo.

Reply:
@tumb1er  thanks. Yup. makes sense, let me add a comment there, but I don't have a detailed context on the reasons behind this.

Reply:
Just for your information: this scripts runs a test suite that generates coverage file on the same code base that checks this file, so first run may fail after codebase changes, and the second one should be fixed (in theory at least).